.schema/config.0.1.json

```diff
@@ -204,8 +209,12 @@
       "additionalProperties": false
     },
     "capabilities": {
+      "type": "object",
+      "additionalProperties": { "$ref": "#/definitions/capability" }
+    },
+    "capabilitiesArray": {
       "type": "array",
-      "items": { "$ref": "#/definitions/capability" }
+      "items": { "$ref": "#/definitions/namedCapability" }

```

Comment:
Does this mean capabilities must have a name even when listed as an array?

Reply:
The name field in `#/definitions/namedCapability` is not a required field.
The effect of this change is that `name` appears in a list of available properties when using array syntax in the IaC file for capabilities. The `name` field *does not* appear when using the map syntax.